cargo-dist/src/tasks.rs

```diff
@@ -2218,6 +2290,9 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             // Always add the source tarball
             self.add_source_tarball(&announcing.tag, release);
 
+            // Add any extra artifacts defined in the config
+            self.add_extra_artifacts(release);
+
```

Comment:
wait so it's obligatorily-workspace-global but added to every Release anyway? At that point feels like it should be per-packageable...

Reply:
I can update the config to try allowing it per-package.